bokehjs/src/coffee/core/ui_events.coffee

```diff
@@ -4,12 +4,14 @@ import {Events} from "./events"
 import {logger} from "./logging"
 import {offset} from "./dom"
 import {getDeltaY} from "./util/wheel"
+import {HammerEvent, PointEvent, KeyDown} from "./bokeh_events"
+
 
 export class UIEvents
   @prototype extends Events
 
   # new (toolbar: Toolbar, hit_area: Element)
-  constructor: (@toolbar, @hit_area) ->
+  constructor: (@toolbar, @hit_area, @event_manager, @plot) ->
```

Comment:
(If I accidentally pinged GitHub use [@plot](https://github.com/plot), sorry that was a mistake!)